### DISCUSSION TITLE
Automatically uploading documentation to unimath.github.io whenever there's a new release

### DISCUSSION THREAD

**Discussion by nmvdw** [UniMath/UniMath#1686] [General]
In papers, it might be more convenient to link to specific definitions, and it tends to be nicer to link to the documentation instead of the code. I am wondering whether it is possible to automatically upload a new version of the UniMath documentation to unimath.github.io whenever we make a new release.

**Comment by benediktahrens** [comment]
UniMath documentation is very big, and any repository will run full very quickly if we upload documentation there regularly. 
In my opinion, it was a mistake to use unimath.github.io for documentation, and I suggest we do not upload any new docs there.

Instead, I suggest people store documentation they would like to have under their own github account, or somewhere else entirely.

**Comment by benediktahrens** [comment]
@nmvdw : have you uploaded a documentation of the Zenodo-indexed release somewhere? Possibly @Kfwullaert and @rmatthes and I would like to link to it, if possible.

**Reply by nmvdw** [reply]
I put it here: https://github.com/nmvdw/FormalTheoryOfMonadsUnivalently

**Reply by rmatthes** [reply]
What is the public URL then?

**Reply by nmvdw** [reply]
For the public base URL, I used:
```
\newcommand{\coqdocbasebaseurl}{https://nmvdw.github.io/FormalTheoryOfMonadsUnivalently}
```
